### DISCUSSION TITLE
Want to contribute french translation

### DISCUSSION THREAD

**Discussion by erreur404** [astubenbord/paperless-mobile#98] [General]
Hi :)

I'd like to contribute the French translation. I understand both German and English as input languages

**Comment by astubenbord** [comment]
Hi, great! What mail can I use to invite you?

**Reply by astubenbord** [reply]
@erreur404 I moved the translations to crowdin over at https://crowdin.com/project/paperless-mobile.

**Reply by JigSawFr** [reply]
> @erreur404 I moved the translations to crowdin over at https://crowdin.com/project/paperless-mobile.

Project seems to be suspended at the moment

**Reply by astubenbord** [reply]
> > @erreur404 I moved the translations to crowdin over at https://crowdin.com/project/paperless-mobile.
> 
> Project seems to be suspended at the moment

Not really suspended but I just currently don't have much time to work on issues.

**Reply by JigSawFr** [reply]
I was speaking about crowdin project sorry, we are unable to participate.
I sent you more infos on Discord ;)

**Reply by astubenbord** [reply]
Hey guys, the problem is now resolved, you should be able to translate again.
